### ISSUE TITLE
Commands: Drop Database

### ISSUE THREAD

**Issue by bricelam** [aspnet/EntityFramework#2476]
In the past, we've talked about having a command to delete the database (i.e. call `EnsureDeleted`). Something like...
```cmd
dnx ef database delete
```

**Comment by benmillane** [comment]
Would it be possible to have options similar to what we've had in the EF SetInitializer method e.g. DropCreateDatabaseAlways or DropCreateDatabaseIfModelChanges. Something like

`dnx . ef migration add NewMigration 
 dnx . ef migration apply -db DropCreateDB`

Not sure on the limitations of the commandline stuff as i'm new to it, but enjoying it so far, just feels like it's lacking in power currently.

**Comment by bricelam** [comment]
`dotnet ef database drop` was added by @laskoviymishka in PR #4657.

I attempted to add `Drop-Database` in PR #4907, but ran into issues marshalling script blocks across domain boundaries.

To complete this issue, we need to...

1. Add a way to get the database name and server from PowerShell (e.g. a `Get-DbContext` command)
2. Add the `Drop-Database` command. The code in [bricelam:fist](https://github.com/aspnet/EntityFramework/compare/dev...bricelam:fist) is a good starting point.
